projects/etcd-io/etcd/1-20/GOLANG_VERSION

```diff
@@ -1 +1 @@
-1.15
+1.16
```

Comment:
Is it usual for them to build Go versions between patch versions?

Reply:
It is unusual. But it looks like they had a failed build process that forced this: https://github.com/etcd-io/etcd/commit/1abf085cfb3a507f1a3d38c7ac7b1e173e1aee33